pyspecdata/core.py

```diff
@@ -7536,6 +7537,10 @@ def _init_datanode(self, datanode, **kwargs):
                         + explain_error(e)
                     )
                 recordarrayofaxis = datadict["axes"][axisname]["data"]
+                if "axis_coords_units" in datadict["axes"][axisname].keys():
+                    myaxis_units[axisnumber] = datadict["axes"][axisname].__getitem__("axis_coords_units")
+                else:
+                    myaxis_units[axisnumber] = None
```

Comment:
If you do what I'm saying above, you don't need this

Reply:
yes I nixed the else